python/tests/core/test_external_feature_group_engine.py

```diff
@@ -35,13 +36,13 @@ def test_save(self, mocker):
 
         f = feature.Feature(name="f", type="str")
 
-        fg = feature_group.FeatureGroup(
+        fg = feature_group.ExternalFeatureGroup(
```

Comment:
Changing in existing tests as it seems this should be ExternalFeatureGroup instead of FeatureGroup